docs/subsystems/typing-indicators.md

```diff
@@ -170,8 +175,13 @@ not let the notifications become too "sticky" either.
 
 ## Roadmap
 
-An area for refinement is to tune the timing values a bit.
-Right now, we are possibly too aggressive about sending `stop`
-messages when users are just pausing to think. It's possible
-to better account for typing speed or other heuristic things
-like how much of the message has already been typed.
+Typing indicators have been implemented for both direct message and
+channel conversations. Future improvements could focus on:
+
+1. Optimizing performance for large channels.
+2. Enhancing the user experience, such as better handling of multiple
+   simultaneous typists in a channel.
+3. Improving the heuristics for when to send "stop" notifications,
+   taking into account factors like typing speed and message length.
+4. Exploring additional customization options for users regarding
+   typing indicators.
```

Comment:
Other than 3, I can't verify where the other items come from and if they are indeed on our roadmap. Could you double-check?

Reply:
yeah I did double checked , there aren't so removing them and just keeping the 3rd point would work

Reply:
Yes that should be fine.

Reply:
Hey @PIG208 Please check the current changes and let me know